api/WorkerLocation.json

```diff
@@ -256,7 +256,7 @@
               "version_added": true
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "14"
```

Comment:
This is the only outlier data in the PR, all the others being Edge 12 + IE 10, but `origin` is a more recent addition than most of `location` which is ancient.

Confirmed with https://staging-dot-mdn-bcd-collector.appspot.com/tests/api/WorkerLocation with https://github.com/foolip/mdn-bcd-collector/pull/1211 deployed that this wasn't in Edge 13 but was in Edge 14.